src/components/Breadcrumbs/BreadcrumbItem.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import BreadcrumbItemStyled from "./BreadCrumbItemStyled";
```

Comment:
Tutaj niepotrzebnie `C` z wielkiej litery - na windows to nie problem, ale na linuxie (serwerze) już rzuca błędem :)